### ISSUE TITLE
54540853 - NHL 2K10

### ISSUE THREAD

**Issue by Makemyday28** [xenia-project/game-compatibility#1615]
[Marketplace](https://marketplace.xbox.com/en-US/Product/NHL-2K10/66acd000-77fe-1000-9115-d80254540853#)

Tested on https://github.com/xenia-project/xenia/commit/ee784fe3901589f29b837197e108533f19c83a39

# Issues:
Cannot create a save game on Dummy HDD, save fails.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5093535/xenia.zip)

# Screenshot(s):
![nhl2k10](https://user-images.githubusercontent.com/69879195/90581550-8b356680-e199-11ea-9789-f7635899382f.JPG)

# Labels:
(recommended labels)

**Comment by Hypno7767** [comment]
NHL 13 which I recently tested upon first boot required you to spam after it said "loading failed" and then it loads. I know this sounds very stupid but did you try spamming or do logs just prove it doesn't work

**Comment by Makemyday28** [comment]
> NHL 13 which I recently tested upon first boot required you to spam after it said "loading failed" and then it loads. I know this sounds very stupid but did you try spamming or do logs just prove it doesn't work

Hello, yes I did spam it and it did load. However, while it would load the game after I spammed the A button, I was not able to make any saves or load any saves further in the game, with respect to the franchise mode and the rosters.

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e

Watched a full CPU vs CPU game. 10-95FPS with vsync off and shaders cached.
:warning:The game can not create a save on Dummy HDD. Just keep pressing A to bypass. Props to @Hypno7767 for the workaround!

![nhlten (5)](https://user-images.githubusercontent.com/31154606/105609967-62318700-5d72-11eb-87c0-f3e66b70e6a3.png)

<details>
  <summary>More Screenshots</summary>

![nhlten (1)](https://user-images.githubusercontent.com/31154606/105609973-68bffe80-5d72-11eb-9f55-8b608ed96560.png)
![nhlten (2)](https://user-images.githubusercontent.com/31154606/105609974-69589500-5d72-11eb-8ddf-256905068d09.png)
![nhlten (3)](https://user-images.githubusercontent.com/31154606/105609976-69f12b80-5d72-11eb-991b-22dde3c4a5ea.png)
![nhlten (4)](https://user-images.githubusercontent.com/31154606/105609977-6a89c200-5d72-11eb-8e2b-227a459b260c.png)
![nhlten (6)](https://user-images.githubusercontent.com/31154606/105609978-6a89c200-5d72-11eb-9df2-9046e5e38496.png)
![nhlten (7)](https://user-images.githubusercontent.com/31154606/105609979-6b225880-5d72-11eb-85f4-1a93929cc318.png)
![nhlten (8)](https://user-images.githubusercontent.com/31154606/105609981-6b225880-5d72-11eb-9079-a49c59470e81.png)
![nhlten (9)](https://user-images.githubusercontent.com/31154606/105609982-6bbaef00-5d72-11eb-90a0-936abd448cdc.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5860681/xenia.zip)
[xenia.z01.zip](https://github.com/xenia-project/game-compatibility/files/5860683/xenia.z01.zip)
[xenia.z02.zip](https://github.com/xenia-project/game-compatibility/files/5860684/xenia.z02.zip)
[xenia.z03.zip](https://github.com/xenia-project/game-compatibility/files/5860685/xenia.z03.zip)
[xenia.z04.zip](https://github.com/xenia-project/game-compatibility/files/5860680/xenia.z04.zip)

state-gameplay

**Comment by Makemyday28** [comment]
Yes I understand to keep pressing A to bypass, I did this however the game will not save in either season mode or franchise mode. The game works fine for playing one match however it does not save in other game modes.

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/ddee85f0be00b3311a258a5ac3faee5e32b91de2

Started Franchise Mode and played some of a Quick Play Match. Saves work.

![nhl (1)](https://user-images.githubusercontent.com/31154606/124043545-d848b280-d9d0-11eb-818b-f07708e23413.png)
![nhl (2)](https://user-images.githubusercontent.com/31154606/124043548-d979df80-d9d0-11eb-8446-54cf81c7b912.png)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/6744460/xenia.zip)

state-gameplay
